:fire: remove camunda registration backend (#6189)
open-formulieren/open-forms: docs/configuration/dmn/camunda.rst

.. _configuration_dmn_camunda:

=========
Camunda 7
=========

> Camunda_ is a process engine using BPM models. Open Forms supports Camunda 7.x

Reviewer: ```suggestion
Camunda_ is a process orchestrator with a DMN engine. Open Forms supports Camunda 7.x
```

Author: Oops I merged before I saw your remark, I'll make a separate PR for it

List process definitions        Open Forms must be able to read the available process definitions to connect a form to a process.
Start process instances         Open Forms must be able to start process instances of the selected process definition(s).
==============================  =======================================================================================

What does the service provider need?
====================================

The values for these parameters should be provided to the service provider by
the Open Forms administrator.

============================  =======================================================================================
Parameter                     Description
============================  =======================================================================================
**Security**
IP address                    The IP address of the Open Forms server (optional, for whitelisting).
============================  =======================================================================================

Plugin configuration
====================

1. In the Open Forms admin, navigate to: **Configuration** > **Camunda configuration**

> 2. In the **Camunda root** field, fill out the protocol and domein (and optional port)

Reviewer 2: ```suggestion
2. In the **Camunda root** field, fill out the protocol and domain (and optional port)
```

   of the API root.
3. In the **REST api path** field, fill out the path of the REST API root, relative to
   the Camunda root URL.
4. Under **Auth**, fill out the username and password. Note that the password is never
   displayed, even if filled out correctly.

The configuration page tests the connection, if any mistakes are made, you will receive

> validation errors.

Reviewer 2: ```suggestion
The configuration page tests the connection, if any mistakes are made, validation errors will be shown.
```
